docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -197,8 +197,9 @@ quarkus.oidc.credentials.jwt.key-file=privateKey.pem
 ----
 quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus/
 quarkus.oidc.client-id=quarkus-app
-quarkus.oidc.credentials.jwt.key-store-file=keystore.jks
-quarkus.oidc.credentials.jwt.key-store-password=mypassword
+quarkus.oidc-client.credentials.jwt.tls-configuration-name=oidc-client
```

Comment:
In this case it looks like an extra property to set without any advantage. IMHO having an option to refer to the specific TLS configuration is great, but it should not be a prerequisite if there is no intention to have a given TLS configuration shared

Reply:
That's correct @sberyozkin . But isn't this always true? If every extensions allows to configure extension specific properties, you will avoid one extra property because you don't need to add link between such extension and TLS registry. So was my thinking to deprecate it. But it is not important to do that.

Reply:
one more question, if we keep both, should I add new example as alternative? I think it might be bit confusing to have them alongside.

Reply:
meaning docs examples one next to another

Reply:
hmm, I can add whole separate example like "or you can do this", so it should be an issue.

Reply:
I would recommend using the tls-configuraiton-name to follow the ADR. OTherwise, your default configuration is going to also be used for the HTTP server, which is unlikely what you want (sharing between the HTTP server and OIDC)